slack_sdk/web/client.py

```diff
@@ -2903,6 +2903,21 @@ def conversations_declineSharedInvite(
         kwargs.update({"invite_id": invite_id, "target_team": target_team})
         return self.api_call("conversations.declineSharedInvite", http_verb="GET", params=kwargs)
 
+    def conversations_externalInvitePermissions_set(
+        self, *, action: str, channel: str, target_team: str, **kwargs
+    ) -> SlackResponse:
+        """Sets a team in a shared External Limited channel to a shared Slack Connect channel or vice versa.
+        https://api.slack.com/methods/conversations.externalInvitePermissions.set
+        """
+        kwargs.update(
+            {
+                "action": action,
+                "channel": channel,
+                "target_team": target_team,
+            }
+        )
+        return self.api_call("conversations.externalInvitePermissions.set", http_verb="GET", params=kwargs)
```

Comment:
Indeed the document says GET method is preferred, but let's go with the default POST here. GET is just the default one on the document site. I don't believe there is no specific reason for selecting GET for this endpoint because it does not make sense for the operation.
```suggestion
        return self.api_call("conversations.externalInvitePermissions.set", params=kwargs)
```

Reply:
Agree, I also thought a GET request did not make sense for this operation but did not know we could substitute it with a POST
TIL